### PR TITLE
included MapUrlTile usage in README.md

### PR DESCRIPTION
**Pull request by ochanje210**

### CODE REVIEW

**Comment by nlpenguin** [comment]
Lgtm!

File: example/examples/LoadingMap.js [hunk 1]
```diff
@@ -41,8 +41,8 @@ class LoadingMap extends React.Component {
           initialRegion={this.state.region}
           onPress={this.onMapPress}
           loadingEnabled
-          loadingIndicatorColor={"#666666"}
-          loadingBackgroundColor={"#eeeeee"}
+          loadingIndicatorColor={'#666666'}
+          loadingBackgroundColor={'#eeeeee'}
         >
```
Comment (christopherdro) [inline example/examples/LoadingMap.js]: See comment above.

File: example/examples/CachedMap.js [hunk 2]
```diff
@@ -66,8 +66,8 @@ class CachedMap extends React.Component {
                 cacheEnabled={this.state.cache}
                 zoomEnabled
                 scrollingEnabled
-                loadingIndicatorColor={"#666666"}
-                loadingBackgroundColor={"#eeeeee"}
+                loadingIndicatorColor={'#666666'}
+                loadingBackgroundColor={'#eeeeee'}
               >
```
Comment (ochanje210) [inline example/examples/CachedMap.js]: I appreciate your advise. I changed back to double quotes and removed curly braces

File: example/examples/LoadingMap.js (hunk 1 above)
Comment (ochanje210) [inline example/examples/LoadingMap.js]: please see comment above

**Comment by spikebrehm** [comment]
Thank you!

**Pull request merged**